Sources/SwiftParser/RawTokenKindSubset.swift

```diff
@@ -612,6 +612,26 @@ enum AwaitTryMove: RawTokenKindSubset {
   }
 }
 
+enum IfOrSwitch: RawTokenKindSubset {
+  case ifKeyword
+  case switchKeyword
+
+  init?(lexeme: Lexer.Lexeme) {
+    switch lexeme {
+    case RawTokenKindMatch(.keyword(.if)): self = .ifKeyword
+    case RawTokenKindMatch(.keyword(.switch)): self = .switchKeyword
```

Comment:
This can be slightly shorter

```suggestion
    case RawTokenKindMatch(.if): self = .ifKeyword
    case RawTokenKindMatch(.switch): self = .switchKeyword
```